### DISCUSSION TITLE
质押矿池未显示在质押列表中

### DISCUSSION THREAD

**Discussion by zoukz115** [mangodager/smartx-dotnet#20] [General]
矿池地址:gGW3cWRZQMCpw6JArhwcRbYrvs3adDuix
该矿池一直在出块，但是却未显示在质押列表中，钱包APP和开发者工具都看不到

**Comment by mangodager** [comment]
This address is in the pledge list
![微信图片_20220622193640](https://user-images.githubusercontent.com/46709419/175020003-2d51b60a-5d87-4644-bb9b-57d12c2b20ff.png)
